libs/sdk-common/src/invoice.rs

```diff
@@ -133,8 +133,10 @@ impl From<lightning::offers::offer::Amount> for Amount {
 
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
 pub struct LNOffer {
+    /// String representation of the Bolt12 offer
     pub bolt12: String,
```

Comment:
Maybe `offer` is more understandable?

Reply:
Done: 643928e4cab2aa0a77f9df5b1c202706181fe293